package.json

```diff
@@ -22,7 +22,7 @@
     "tsc:esm": "tsc -p tsconfig.json",
     "tsc:cjs": "tsc -p tsconfig-cjs.json",
     "build": "rm -rf lib && npm run compile",
-    "build:readme": ". ./scripts/markdown-magic/build-readme.sh",
+    "build:readme": "bash ./scripts/markdown-magic/build-readme.sh",
```

Comment:
Didn't mean to commit this but probably better that it's there, script won't run right on Windows without it